xclim/analog.py

```diff
@@ -289,7 +286,7 @@ def zech_aslan(x: np.ndarray, y: np.ndarray, *, dmin: float = 1e-12) -> float:
     :math:`r \to 0`, it is controllable through the `dmin` parameter.
 
     This version corresponds the :math:`D_{ZAE}` test of :cite:t:`grenier_assessment_2013` (eq. 7), which is a version
-    of :math:`\phi_{NM}` from :cite:t:`aslan_new_2003`, modified by using the standardized  euclidean distance, the log
+    of :math:`\phi_{NM}` from :cite:t:`aslan_new_2003`, modified by using the standardized Euclidean distance, the log
     weight function and choosing :math:`d_{min} = 10^{-12}`.
```

Comment:
Do we want Ruff to manage this kind of stuff?

Reply:
Oh, that was me seeing a typo. Ruff isn't that advanced! haha